### PR TITLE
Add centos6 configuration

### PR DESCRIPTION
**Pull request by dhodovsk**

### CODE REVIEW

File: distconf/centos-6-x86_64.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+extends: "lib/centos.yaml"
+
+os:
+  version: 6
+  arch: x86_64
+
+docker:
+  from: centos:centos6
```
Comment (TomasTomecek) [inline distconf/centos-6-x86_64.yaml]: registry.centos.org or docker.io, votes?

Comment (dhodovsk) [reply distconf/centos-6-x86_64.yaml]: This value is now in all sclorg's repositories set to centos:centos6, we might consider adding configurations on different base image registries in the future.

**Comment by praiskup** [comment]
Could we also add rhel-8-x86_64 ? :-)

**Comment by praiskup** [comment]
RHEL-8 is in different PR (and this is about centos anyway).

Ok, lgtm, can we mention this new config in some testcase at least?  Updating some `distros` file under `tests/` should be enough.

**Comment by praiskup** [comment]
The CI got fixed in the meantime, please rebase.

**Comment by praiskup** [comment]
Thanks!

**Pull request merged**